scripts/generate-stackbrew-library.js

```diff
@@ -106,17 +104,17 @@ function generateOsArchLibrary (context) {
     context.imageName
   )
 
-  tags.forEach((tag) => {
-    content += `${tag}: ${URL}@${commit.slice(0, -1)} ${path.join(ROOT, context.path)}\n`
-  })
+  const content = {
+    tag: tags[0],
+    repoDir: `${URL}@${commit.slice(0, -1)} ${path.join(ROOT, context.path)}`,
+    alias: tags.join(' ')
```

Comment:
You should leave the tags as an array here and use `jq` in bashbrew to iterate on them

Reply:
@petrosagg, I don't want to add more unnecessary complex logic to the bashbrew script since image tag only contains any letter, dot or dash so put all the alias tags in a string with space delimiter then we can iterate on them simple as this https://github.com/balena-io-library/official-images/pull/235/files#diff-015d29779976b02f9d9008807a990f07R72

Reply:
I don't think it's unecessary logic nor that it's complex.

Actually I just realised you're using string functions to "parse" the JSON like `firstTag="${repoTag#*:}"`. This is not the right way to do thing, even if in this particular occasion it works so we should fix those too.

It's very simple to use `jq` from a bash script to properly interact with structured data and it doesn't make sense to compromise the data representation in order to use string functions to parse it. For example to get the repoDir all you need to do is `echo $json_data | jq .repoDir`

Reply:
@petrosagg we do use `jq` to parse the json first here https://github.com/balena-io-library/official-images/blob/08cecfd46f5e0fec2620c9f0a0a19cca2cdd0efb/bashbrew/bashbrew.sh#L313-L315. `firstTag="${repoTag#*:}"` as you see is just string manipulation, not JSON parsing.